app/bundles/InstallBundle/Tests/InstallFixtures/ORM/GrapeJsDataTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright   2021 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        https://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\InstallBundle\Tests\InstallFixtures\ORM;
+
+use Mautic\CoreBundle\Test\MauticMysqlTestCase;
+use Mautic\InstallBundle\InstallFixtures\ORM\GrapesJsData;
+use Mautic\PluginBundle\Entity\Integration;
+use Mautic\PluginBundle\Entity\Plugin;
+use PHPUnit\Framework\Assert;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class GrapeJsDataTest extends MauticMysqlTestCase
+{
+    use FakeContainerTrait;
+
+    protected $useCleanupRollback = false;
+
+    private GrapesJsData $fixture;
+
+    protected ContainerInterface $tempContainer;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->tempContainer = self::$container;
+        $this->fixture       = new GrapesJsData();
+        $this->fixture->setContainer($this->getContainerFake());
+    }
+
+    public function testGetGroups(): void
+    {
+        Assert::assertSame(['group_install', 'group_mautic_install_data'], GrapesJsData::getGroups());
+    }
+
+    public function testGetOrder(): void
+    {
+        Assert::assertSame(1, $this->fixture->getOrder());
+    }
+
+    public function testLoad(): void
+    {
+        $findOneByCriteria = [
+            'name'        => 'GrapesJS Builder',
+            'description' => 'GrapesJS Builder with MJML support for Mautic',
+            'version'     => '1.0.0',
```

Comment:
Why is the version important for this test? It will have to be updated every time a new version of the plugin is released, right? Or if the description will change. I'd leave only the bundle name in the test.

Reply:
Agree that we should not be setting the version anywhere (it's now set in the composer file for plugins and themes).  Must have been missed in the code review :(